tests/ipi2/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(ipi2 "psi;quicktests;smoketests;cc")
```

Comment:
external deps tests (e.g., dftd3) usually go in a separate subdir of tests. if you don't want to, I'll reposition the tests after merge. in that case, this is just a heads-up for when you notice the tests aren't where you left them.

Reply:
I put both tests into a subfolder `ipi`. If you prefer another location, feel free to reposition them as you wish. Thank you for the heads-up